pages/api/[entity_type_slug].js

```diff
@@ -114,7 +114,7 @@ async function get(req, res) {
 
     setCORSHeaders({ response: res, url: process.env.FRONTEND_URL });
 
-    rawData ? res.status(OK).json(output ?? []) : res.status(NOT_FOUND).json({});
+    rawData.data.length > 0 ? res.status(OK).json(output ?? []) : res.status(NOT_FOUND).json({});
```

Comment:
This is not always true. It is possible that the `entity_type_slug` really exists, but the client hasn't yet entered any entries. In that scenario, we need to return a `200`, even if there is no data available.

We only return a 404 error when there is no EntityType with that certain slug.